File: 15.Functions - Exercise/04_odd_and_even_sum.py
def numbers(number):
    sum_of_odd_digits = 0
    sum_of_even_digits = 0
    for current_number in number:
        if int(current_number) % 2 == 0:
            sum_of_even_digits += int(current_number)
        else:
            sum_of_odd_digits += int(current_number)
    return sum_of_odd_digits, sum_of_even_digits


user_input = input()
sum_of_odd_digits, sum_of_even_digits = numbers(user_input)
print(f"Odd sum = {sum_of_odd_digits}, Even sum = {sum_of_even_digits}")

File: JS Front End/JS Front End Tasks/06. Functions and Statements - Exercises/04. Odd And Even Sum.js
function oddAndEvenSum(number) {
    let numberAsString = number.toString()
    let oddSum  = 0
    let evenSum  = 0

    for (let i = 0; i < numberAsString.length; i++) {
        if (Number(numberAsString[i]) % 2 === 0) {
            evenSum += Number(numberAsString[i])
        } else {
            oddSum += Number(numberAsString[i])
        }
        
    }

    console.log(`Odd sum = ${oddSum}, Even sum = ${evenSum}`)
}

oddAndEvenSum(1000435)
oddAndEvenSum(3495892137259234)